### ISSUE TITLE
Simplify `enter` / `leave` processing with callback: possible to pass the event type into the callback?

### ISSUE THREAD

**Issue by WhereJuly** [silvestreh/onScreen#50]
Hi Silvestre,

quite a beatiful OnScreen you made :1st_place_medal: 

Could it be possible to pass the event type into the callback invocation? This would have saved us from making 2 callback for 2 events, whereas there could be the only one callback. Though this does not deny 2 callback anyhow.

Now the use case looks like:
  ``` javascript
            var os = new OnScreen();
            os.on('enter', 'footer', cbEnter);
            os.on('leave', 'footer', cbLeave);

            function cbEnter(elt) {
                console.log('process `enter` event'); 
                addClass(elt, 'onScreen');
                console.log(elt + " enters the screen");
                // etc.
            }

            function cbLeave(elt) {
                console.log('process `leave` event'); 
                removeClass(elt, 'onScreen');
                console.log(elt + " leaves the screen");
                // etc.
            }
  ```

With `event name` available from within callback it could look a bit shorter and easier to read:
  ``` javascript
            var os = new OnScreen();
            os.on('enter', 'footer', cbProcessOnScreenEvent);
            os.on('leave', 'footer', cbProcessOnScreenEvent);

            function cbProcessOnScreenEvent(elt, osevt) {
                console.log(); 'process '+ osevt + ' event';
                osevt === `enter` ? elt.addClass('onScreen') : elt.removeClass('onScreen');
                console.log(elt + ' ' + osevt + ' the screen');
            }
  ```
Notice `osevt` event name passed into the callback.
We replace two named callbacks with one, simplify the code a lot, delete repeating parts, and make it more readable / maintainable.

What do you think?

Best regards,
Val

**Comment by silvestreh** [comment]
Added in b3a16f9 … still unreleased though 😃

**Comment by silvestreh** [comment]
aaaaand… it's available on `1.3.0` 🎉

**Comment by WhereJuly** [comment]
Thanks a lot. Will enjoy testing it within a week or so :100:
